### ISSUE TITLE
Improve Keyboard Access

### ISSUE THREAD

**Issue by SteveChab** [cmu-delphi/www-covidcast#188]
- Display Type labels are `button` elements that don't do anything when clicked. Instead, they should be `label` elements. See example HTML below to correct the problem.
- There are two Display type groups with the same `aria-label`. Each group should have a unique label. See example HTML below to correct the problem.
<img width="717" alt="Screen Shot 2020-06-23 at 12 39 12 PM" src="https://user-images.githubusercontent.com/275785/85430780-929c1300-b54e-11ea-8618-799a7ea6b1db.png">
